src/main/java/dev/minguinho/zeze/domain/slide/model/Slide.java

```diff
@@ -22,13 +22,24 @@
     private String content;
     @Enumerated(EnumType.STRING)
     private AccessLevel accessLevel;
+    private Long userId;
+
+    public Slide(String title, String content, AccessLevel accessLevel) {
+        this.title = title;
+        this.content = content;
+        this.accessLevel = accessLevel;
+    }
 
     public void update(Slide slide) {
         this.title = slide.title;
         this.content = slide.content;
         this.accessLevel = slide.accessLevel;
     }
 
+    public boolean isNotOwner(Long userId) {
+        return !this.userId.equals(userId);
+    }
+
```

Comment:
```suggestion
    public boolean isOwner(Long userId) {
        return this.userId.equals(userId);
    }
```

메소드에 부정형을 붙이는 것보단 긍정형으로 바꾸고 사용시에 `!`를 사용하는 것이 더 좋아보여요.